index.html

```diff
@@ -248,51 +186,89 @@ <h3><dfn>Sampling Frequency</dfn> and <dfn>Reporting Frequency</dfn></h3>
   </p>
 </section>
 
+<section> <h2>Pressure States</h2>
+  <p>
+    <dfn>Pressure states</dfn> represents the minimal set of useful states that allows web applications
```

Comment:
Please use `websites` (there and below) instead of `web applications` for consistency.

Reply:
fixed

Reply:
Use `website`, not `web site` ;)